src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java

```diff
@@ -0,0 +1,214 @@
+/**
```

Comment:
@proshin-roman Again, 13 files. To quote Donald Trump: "Yuge PR". Make more PRs, please, Or you can add puzzles for future PRs, however you wish. For now, in the scope of this PR, I would only accept the 2 test files above.